.github/workflows/release_workflow.yml

```diff
@@ -86,6 +86,27 @@ jobs:
             pom.xml
             linux-deployment/*.deb
             
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
```

Comment:
where are these secrets coming from? if they are stored in Github repo, can you send the secrets to me via slack dm?